src/Pages/ProductList/Components/Option.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+class Option extends Component {
+  render() {
+    return (
+      <option value={this.props.item.value}>{this.props.item.text}</option>
+    );
+  }
```

Comment:
구조분해 사용하면 줄일 수 있습니다!

```suggestion
  render() {
    const {item} = this.props;
    return (
      <option value={item.value}>{item.text}</option>
    );
  }
```

Reply:
넵 !! 감사합니다 !!